node/src/BaseClient.ts

```diff
@@ -1194,6 +1198,34 @@ export class BaseClient {
         return this.createWritePromise(createZremRangeByRank(key, start, end));
     }
 
+    /**
+     * Adds an entry to the specified stream.
+     * See https://redis.io/commands/xadd/ for more details.
+     *
+     * @param key - The key of the stream.
+     * @param values - field-value pairs to be added to the entry.
+     * @returns The id of the added entry, or `null` if `options.makeStream` is set to `false` and no stream with the matching `key` exists.
+     */
+    public xadd(
+        key: string,
+        values: [string, string][],
```

Comment:
why not using Record? it doesn't keep order?

Reply:
In javascript it keeps the order of insertions, but not in C#, python or others.